doc/application/application.rst

```diff
@@ -935,155 +935,91 @@ The example below shows a comment line and an override setting
 
-      A question-based menu opens that allows you to set individual
-      configuration options.
+#. Set kernel configuration values as follows:
 
-      .. image:: figures/app_kernel_conf_1.png
-           :width: 600px
-           :align: center
-           :alt: Main Configuration Menu
+   * Use the arrow keys to navigate the menu.
 
-#. Set kernel configuration values using the following
-   key commands:
+     .. note::
 
-   * Use the arrow keys to navigate within any menu or list.
+        Common `Vim <https://www.vim.org>`_ navigation commands are
+        supported as well.
 
-   * Press :kbd:`Enter` to enter submenus and choices, which appear with
-     ``--->`` next to them.
+   * Press :kbd:`Enter` or :kbd:`Space` to enter submenus and choices, which
+     appear with ``--->`` next to them. Press :kbd:`ESC` to return to the
+     parent menu.
 
    * Press :kbd:`Space` to toggle or configure a symbol value. Boolean
      configuration symbols are shown with :guilabel:`[ ]` brackets, while
      numeric and string-valued configuration symbols are shown with
      :guilabel:`( )` brackets.
 
-   * Press :kbd:`Tab` to navigate the command menu at the bottom of the
-     display.
-
-#. For information about any symbol, select the symbol and type :kbd:`?`.
+#. To view information about a symbol, highlight it and type :kbd:`?`.
 
-   Press :kbd:`Enter` to return to the menu.
+   Pressing :kbd:`ESC` or :kbd:`Q` returns from the information display to the
+   menu display.
 
-#. Press :kbd:`/` to bring up a search menu to look for a particular option.
-
-#. After configuring the kernel options for your application, tab to
-   :guilabel:`< Save >` and press :kbd:`Enter`.
-
-   The following dialog opens with the :guilabel:`< Ok >` command selected:
+#. After configuring the kernel options for your application, press
+   :kbd:`Q` to bring up the save-and-quit dialog:
 
    .. image:: figures/app_kernel_conf_2.png
-      :width: 400px
       :align: center
-      :height: 100px
-      :alt: Save Configuration Dialog
-
+      :alt: Save and Quit Dialog
 
-#. Press :kbd:`Enter` to save the kernel configuration options to the default
-   file name; alternatively, type a file name and press :kbd:`Enter`.
+#. Press :kbd:`Y` to save the kernel configuration options to the default file
+   name (:file:`zephyr/.config`).
 
    Typically, you will save to the default file name unless you are
    experimenting with various configuration scenarios.
 
-   A :file:`zephyr` directory will have been created in the build
-    directory.
-
    .. note::
 
-      At present, only a :file:`.config` file can be built. If you have saved
-      files with different file names and want to build with one of these,
-      change the file name to :file:`.config`. To keep your original
-      :file:`.config`, rename it to something other than :file:`.config`.
-
-   Kernel configuration files, such as the :file:`.config` file, are saved
```

Comment:
Why did you remove this paragraph? Seems to still apply and there's valuable info there.

Reply:
It wasn't removed, just dedented. See below. :)

Reply:
Ah, sorry, I see which paragraph you mean now.

I wonder how helpful it is in practice to explain how to list files starting with `.` here. The description is *nix-specific as well, as those files are always shown on Windows.

Maybe a short note along the lines of `Note that 'ls' does not list files starting with '.' by default. You can pass the '-a' flag to see them.` would be enough, if we keep it.

Reply:
yes, maybe something like "The Zephyr configuration file generated during CMake configuration is stored in `zephyr/.config`, and is not visible by default on Linux and macOS. Use `ls -a` to be able to see it"